.github/workflows/test.yml

```diff
@@ -0,0 +1,52 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: build (Python ${{ matrix.python-version }}, Django ${{ matrix.django-version }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 5
+      matrix:
+        python-version: ['3.6', '3.7', '3.8', '3.9']
+        django-version: ['2.2', '3.0', '3.1', 'master']
```

Comment:
Nice 👍. Very clever to get tox to manage the complexity of installing Django master.

Reply:
Yeah, there are a few ways to do it, but seems quite flexible.